### ISSUE TITLE
Doc may indicates to strip binaries

### ISSUE THREAD

**Issue by guimard** [rumpkernel/rumprun#63]
Hi,
following wiki example, I've build a 33M image of nginx. Using simply `strip nginx.bin`, size is reduced to 5M.
Thanks for this big work !

**Comment by anttikantee** [comment]
Yes, I agree, but I'm not sure *where* to mention that.  Do you have a suggestion?  We could also talk about further reducing binary size by eliminating unnecessary components from the `rumprun-bake` configuration.

Furthermore, `strip nginx.bin` is, strictly speaking, incorrect.  You should use `foo-rumprun-netbsd(elf)-strip`.
